tests/test_api.py

```diff
@@ -250,66 +268,48 @@ async def test_api_get_resource_status(
     assert data["latest_check_url"].endswith(f"/api/checks/latest?resource_id={RESOURCE_ID}")
 
 
-@pytest.mark.parametrize(
-    "route",
-    [
-        {"url": "/api/resources/", "method": "post"},
-        {"url": "/api/resource/created/", "method": "post"},  # legacy route
-    ],
-)  # TODO: can be removed once we don't use legacy route anymore
 async def test_api_create_resource(
-    client, route, api_headers, api_headers_wrong_token, udata_resource_payload
+    client, api_headers, api_headers_wrong_token, udata_resource_payload
 ):
-    # TODO: can be removed once we don't use legacy route anymore
-    client_http_method: Callable = getattr(client, route["method"])
-
     # Test API call with no token
-    resp = await client_http_method(path=route["url"], headers=None, json=udata_resource_payload)
+    resp = await client.post(path="/api/resources/", headers=None, json=udata_resource_payload)
```

Comment:
Yes, it's on purpose. In a previous merged PR, we added both the trailing slash option and no trailing slash option for each route (see routes/__init__.py) in oder not not have to thin about this anymore.

Since it's now done, I refactored the tests to make the calls with no trailing slash in this present PR, which is a bit more standard to me.

Reply:
See https://github.com/datagouv/hydra/pull/132/files#diff-e5ce70c8c6e9d9d078789d97ab169134b487c04923fa2b5b8bc584edee548e06R22